tests/ExampleApps/PickingDemoApp/CMakeLists.txt

```diff
@@ -0,0 +1,79 @@
+cmake_minimum_required(VERSION 3.6)
+if (${CMAKE_MAJOR_VERSION}.${CMAKE_MINOR_VERSION} VERSION_GREATER "3.9")
+    cmake_policy(SET CMP0071 NEW)
+endif ()
+if (APPLE)
+    cmake_policy(SET CMP0042 NEW)
+endif (APPLE)
+
+set(CMAKE_DISABLE_SOURCE_CHANGES ON)
+set(CMAKE_DISABLE_IN_SOURCE_BUILD ON)
+
+project(PickingDemoApp VERSION 0.0.1)
+
+if (NOT CMAKE_BUILD_TYPE)
+    set(CMAKE_BUILD_TYPE Release)
+endif ()
+
+# Set default install location to installed-<Compiler_ID> folder in build dir
+# we do not want to install to /usr by default
+if (CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT)
```

Comment:
This change the main `CMAKE_INSTALL_PREFIX` on a fresh cmake configuration
(the value `CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT` do not change its value when `set(CMAKE_INSTALL_PREFIX` ...) is called in main `CMakeLists.txt`

Reply:
Resolved by fffaeb9